cmd/neofs-node/config.go

```diff
@@ -307,8 +307,8 @@ type internals struct {
 	log *logger.Logger
 
 	wg      *sync.WaitGroup
-	workers []worker
-	closers []func()
+	workers []*worker
```

Comment:
Why do we need to use slice of pointers? It almost always unnecessary. Same for `closers`

Reply:
Reverted